examples/todo/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "mud-example-minimal",
```

Comment:
might wanna update this name

also might be a tiny bit more clear if this dir is `todo-app` and the package named after it like `mud-example-todo-app`

(just wanna make sure folks don't think this is a dir of our TODOs haha)

Reply:
lmao true